sys/include/net/gnrc/netif.h

```diff
@@ -41,6 +41,11 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Identifier of the loopback interface
+ */
+#define GNRC_NETIF_LOOPBACK_PID     (-1)
```

Comment:
Actually, -1 was your proposal:
https://github.com/RIOT-OS/RIOT/pull/5247#issuecomment-237845307

*KERNEL_PID_LAST + 1` would work for me, too.